README.md

```diff
@@ -37,7 +37,7 @@ Text::WordDiff requires the following modules:
 * Text::Diff 0.11
 * HTML::Entities
 
-Copyright and Licence
```

Comment:
Bwa-ha-ha-ha!

Reply:
Yeah. I have nothing against the British spelling, but I have this thing about consistency, and everywhere else the US spelling is used. ;)

Reply:
BTW, @n1vux also reviewed this PR before I merged it (even though no comments were left here).